test-utils-base/src/commonMain/kotlin/FlowTest.kt

```diff
@@ -89,6 +89,21 @@ private suspend fun <TC : TestContext> testContext(cookie: Long, testContext: su
 @SharedImmutable
 private val cookieTin = AtomicLong(0L)
 
+abstract class NoConfigurationBaseFlowTest<TC : TestContext, T, F : Flow<T>>(scope: CoroutineScope = MainScope()) :
```

Comment:
I'd name it as BaseFlowTest, while one with configuration rename into BaseFlowTestWithConfiguration.
It will no break existing code (which is not yet migrated into 0.5.0 and also test without configuration is used more ofter I think

Reply:
The downside is to migrate one more time existing running project which are on snapshot already